google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta2/BQTableSchemaToProtoDescriptorTest.java

```diff
@@ -233,6 +233,12 @@ public void testStructComplex() throws Exception {
             .setMode(TableFieldSchema.Mode.NULLABLE)
             .setName("test_numeric_str")
             .build();
+    final TableFieldSchema TEST_NUMERIC_SHORT =
```

Comment:
Since only the v1 client is updated, v1beta2 tests are not needed. Can you please remove this?

Reply:
I can, I wanted to maintain backwards compat, but if thats not an issue I will change